google/cloud/storage/benchmarks/benchmark_make_random_test.cc

```diff
@@ -51,7 +51,8 @@ TEST(StorageBenchmarksUtilsTest, MakeRandomBucket) {
   EXPECT_EQ(0, d1.rfind("prefix-", 0));
   EXPECT_GE(63U, d1.size());
   EXPECT_EQ(std::string::npos,
-            d1.find_first_not_of("-_abcdefghijklmnopqrstuvwxyz012456789"));
+            d1.find_first_not_of("-_abcdefghijklmnopqrstuvwxyz0123456789"))
```

Comment:
You may want to take a look at ...

- google/cloud/storage/examples/storage_examples_common.cc
- google/cloud/storage/internal/curl_client.cc
- google/cloud/storage/internal/generate_message_boundary_test.cc
- google/cloud/storage/testing/random_names.cc
- google/cloud/storage/testing/storage_integration_test.cc
- google/cloud/storage/tests/object_media_integration_test.cc
- google/cloud/storage/tests/object_rewrite_integration_test.cc

too.

Reply:
Thanks, done.  I searched for all strings missing a digit (e.g. `012345689`) and/or missing a letter (e.g. `abcdefhil...`). Let me know if you think of other patterns